web/src/containers/QuarterlyBudgetSummary.js

```diff
@@ -7,150 +7,130 @@ import { t } from '../i18n';
 
 const FUNDING_SOURCES = [['hitAndHie', 'HIT and HIE'], ['mmis', 'MMIS']];
 const QUARTERS = [1, 2, 3, 4];
-const COLORS = ['teal', 'green', 'yellow'];
 const EXPENSE_NAME_DISPLAY = {
   state: t('proposedBudget.quarterlyBudget.expenseNames.state'),
   contractors: t('proposedBudget.quarterlyBudget.expenseNames.contractor'),
   combined: t('proposedBudget.quarterlyBudget.expenseNames.combined')
 };
 
-const color = idx => `bg-${COLORS[idx] || 'gray'}`;
-
 const QuarterlyBudgetSummary = ({ budget, years }) => {
   // wait until budget is loaded
   if (!years.length) return null;
 
   return (
-    <div>
+    <Fragment>
       {FUNDING_SOURCES.map(([source, sourceDisplay]) => {
         const data = budget[source];
         return (
           <div
             key={source}
-            className="mb3 table-frozen-wrapper table-frozen-wide-header"
+            className="mb3"
           >
-            <h3 className="mt0">{sourceDisplay}</h3>
-            <div className="overflow-x table-frozen-scroller">
-              <table
-                className="table-cms table-frozen-left-pane"
-                aria-hidden="true"
-              >
-                <thead>
-                  <tr>
-                    <th className="table-frozen-null-cell">--</th>
-                  </tr>
-                  <tr>
-                    <th className="table-frozen-null-cell">--</th>
-                  </tr>
-                </thead>
-                <tbody>
-                  {Object.keys(EXPENSE_NAME_DISPLAY).map(name => (
-                    <tr
-                      key={name}
-                      className={`${name === 'combined' ? 'bold' : ''}`}
-                    >
-                      <td>{EXPENSE_NAME_DISPLAY[name]}</td>
-                    </tr>
-                  ))}
-                </tbody>
-              </table>
-              <table className="table-cms table-frozen-data">
-                <thead>
-                  <tr>
-                    <th id={`quarterly-budget-summary-${source}-null1`} />
-                    {years.map((year, i) => (
-                      <th
-                        key={year}
-                        className={`center ${color(i)}`}
-                        colSpan="5"
-                        id={`quarterly-budget-summary-${source}-fy-${year}`}
-                      >
-                        {t('ffy', { year })}
-                      </th>
-                    ))}
+            <h3 className="ds-h3">{sourceDisplay}</h3>
+            {years.map((year) => (
+            <table className="table-cms">
+              <colgroup>
+                <col className="table-cms--col-header__fixed-width" />
```

Comment:
Whoa, what is this witchcraft?

![mind blown](https://media.giphy.com/media/d0MYTsbRnGKoU/giphy.gif)

Reply:
Writing ancient HTML is like writing runes, man. (You _could_ just put a class on the actual header, but since it's in a loop, I kind of feel like writing it out as a colgroup makes it a little more readable? Or maybe that's just because I, like HTML, am ancient?)